Translation/DatabaseFreshResource.php

```diff
@@ -2,14 +2,14 @@
 
 namespace Lexik\Bundle\TranslationBundle\Translation;
 
-use Symfony\Component\Config\Resource\ResourceInterface;
+use Symfony\Component\Config\Resource\SelfCheckingResourceInterface;
```

Comment:
In fact others changes (related to forms) must be done to avoid deprecations in Symfony 2.8 or upper. I think you can create a new 4.0.0 branch for Symfony >= 2.8.